app/services/users.js

```diff
@@ -0,0 +1,29 @@
+const bcrypt = require('bcrypt');
```

Comment:
Did this library work for you? Because you have bcryptjs installed and it is different than bcrypt.